upgrading/rosa-upgrading-sts.adoc

```diff
@@ -1,31 +1,42 @@
 
 toc::[]
 
+Use one of the following methods to upgrade ROSA (classic architecture) clusters:
+
+* Manually through the ROSA CLI (`rosa`) - Start a one-time immediate upgrade or schedule a one-time upgrade for a future date or time.
+* Manually through the {cluster-manager} UI - Start a one-time immediate upgrade or schedule a one-time upgrade for a future date or time.
+* Automated upgrades - Set an upgrade window for automatic recurring y-stream upgrades whenever a new version is available. Upgrades to minor versions must be manually scheduled.
+
 [id="rosa-lifecycle-policy_{context}"]
 == Life cycle policies and planning
 
 To plan an upgrade, review the xref:../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[{product-title} update life cycle]. The life cycle page includes release definitions, support and upgrade requirements, installation policy information and life cycle dates.
 
-Upgrades are manually initiated or automatically scheduled. Red{nbsp}Hat Site Reliability Engineers (SREs) monitor upgrade progress and remedy any issues encountered.
+You can use update channels to decide which Red Hat OpenShift Container Platform minor version to update your clusters to. {product-title} only supports updates through the `stable` channel. To learn more about OpenShift update channels and releases, see link:https://docs.openshift.com/container-platform/4.15/updating/understanding_updates/understanding-update-channels-release.html[Understanding update channels and releases].
 
 [id="rosa-sts-upgrading-a-cluster-with-sts"]
 == Upgrading a ROSA Classic cluster
 
-There are two methods to upgrade Classic {product-title} (ROSA) clusters:
-
-* Individual upgrades through the ROSA CLI (`rosa`)
-* Individual upgrades through the {cluster-manager} console
+You must upgrade ROSA (classic architecture) clusters using either the ROSA CLI (`rosa`) or the {cluster-manager} console.
 
 [NOTE]
 ====
-When you follow a scheduled upgrade policy, a delay of an hour or more before the upgrade process begins might occur, even if the upgrade is configured to begin immediately. Additionally, the duration of the upgrade might vary based on your workload configuration.
+The actual start time of the cluster upgrade will be within one hour of the upgrade schedule time. Additionally, the duration of the upgrade might vary based on your workload configuration.
 ====
 
+When a ROSA (classic architecture) cluster that uses AWS Security Token Services (STS) is upgraded, the ROSA CLI verifies the account and operator role policies for the chosen cluster are compatible with the target version of the upgrade. If the policies are compatible, the CLI automatically upgrades the cluster. If the policies are not compatible with the chosen upgrade version, the CLI prompts you to upgrade them. When scheduling the upgrade, you provide administrative acknowledgment to confirm you have reviewed the changes involved with the upgrade, if required.
```

Comment:
Again here, the CLI doesn't prompt users to upgrade them.  https://docs.google.com/document/d/1SQPZNiuWPP7_vQkp6wLR1_Wz45yFc718UQkq_qxRSwU/edit

Reply:
Please change this:
If the policies are not compatible with the chosen upgrade version, the CLI prompts you to upgrade them.
to 
If the policies are not compatible with the chosen upgrade version, the CLI automatically upgrades them.